.gitignore

```diff
@@ -234,3 +234,7 @@ Cargo.lock
 
 # These are backup files generated by rustfmt
 **/*.rs.bk
+
+# The name of our generated ClamAV-Rust bindings file.
+libclamav_rust/src/sys.rs
```

Comment:
I was thinking the same thing. I was tempted to name it `cli`, to mirror the `cli_` prefix we use for internal API that aren't stable for public use... but then I thought maybe we want to use `cli` in the future as a module that provides safe wrappers around `sys` bindings.  

I copied the `sys` name from someone else's work.  I agree that it does sort of imply that it comes from the "system" but since it also represents "the unsafe bindings" vs `rs` which would be used for "safe idiomatic wrapper around sys-bindings" I figure `sys` is good enough.  I at least like that it's shord :D